src/app/courses/courses.component.html

```diff
@@ -1 +1,13 @@
-<crs-course-list [courses] = "coursesMock"></crs-course-list>
\ No newline at end of file
+<div class="search">
+    <input [(ngModel)]="inputValue" type="search" placeholder="Type to search">
+    <button (click)="searchHandler()">Search</button>
+</div>
+<div class="add-control-box">
+    <button class="add-button">Add course</button>
+</div>
+<ng-container *ngIf="!coursesMock.length else courses">
```

Comment:
Hi! It's a better approach to keep business logic on the `*.ts` file.

Try the following:

1. create a `boolean` `get` property in `courses.component.ts`
1. move this logic to the new property
1. user the `get` property here

Reply:
Done